server/src/main/java/org/elasticsearch/action/update/UpdateResponse.java

```diff
@@ -162,8 +162,9 @@ public UpdateResponse build() {
                 update = new UpdateResponse(shardId, type, id, version, result);
             }
             if (getResult != null) {
-                update.setGetResult(new GetResult(update.getIndex(), update.getType(), update.getId(), update.getVersion(),
-                        getResult.isExists(),getResult.internalSourceRef(), getResult.getFields()));
+                update.setGetResult(new GetResult(update.getIndex(), update.getType(), update.getId(),
+                    getResult.getSeqNo(), getResult.getPrimaryTerm(), update.getVersion(),
```

Comment:
Why do we use the seq-no and primary term from `getResult` whereas we use the version from `update`?

Reply:
I saw the too and tried to change it but tests fail. IMO it's a bug but I chose not to fix it in this PR / break BWC.

Reply:
ok